signup-service/src/main/java/io/mosip/signup/dto/ResetPasswordRequest.java

```diff
@@ -12,4 +12,6 @@ public class ResetPasswordRequest {
 
     @Password
     private String password;
+
+    private String locale;
```

Comment:
Do we need validation only if a value is provided in this field? I believe we have language validator annotation already with a required boolean flag